packages/pluggableWidgets/listview-swipe-native/src/ListViewSwipe.editorConfig.ts

```diff
@@ -0,0 +1,45 @@
+import { StructurePreviewProps, RowLayoutProps, ContainerProps } from "@widgets-resources/piw-utils";
+import { ReactNode } from "react";
+import { ListViewSwipeProps } from "../typings/ListViewSwipeProps";
+
+const rowLayoutContainer = (property: ReactNode): ContainerProps => ({
```

Comment:
If i understand correctly, `rowLayoutContainer` is a container around the content of left or right. So for that i would rename this to `contentContainer`. When it is a `rowLayoutContainer`, I assumed that it is the most outer row (layout) container. What do you think @wesleysieses ?

Reply:
I agree, updated!